src/app/components/views/weather-view/weather-view.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+
+import { WeatherAlertResponse } from 'src/app/models/weather-alert.model';
+import { WeatherService } from 'src/app/services/weather.service';
+
+@Component({
+  selector: 'app-weather-view',
+  templateUrl: './weather-view.component.html',
+  styleUrls: ['./weather-view.component.scss']
+})
+export class WeatherViewComponent implements OnInit {
+  public weatherAlertResponse: WeatherAlertResponse;
+
+  constructor(private weatherService: WeatherService) { }
+
+  ngOnInit(): void {
+    this.subscribeToWeatherService();
+}
+
+/**
+ * @description - Subscribes to weather service and sets global variable for the api response
+ * @returns {void}
+ */
+  private subscribeToWeatherService(): void {
+    this.weatherService.getLoading().subscribe({
```

Comment:
change to `this.weatherService.getLoading().pipe( takeUntil( this.ngUnsubscribe )).subscribe({`

we'll also need to create the ngUnsubscribe and some other stuff